core/extraction/round_two.py

```diff
@@ -898,44 +899,95 @@ def extract_footfall_outcomes(df_input: pd.DataFrame) -> pd.DataFrame:
-        ]
-        temp_cols = pd.concat([df_prog_id, temp_cols], axis=1)
-        temp_cols.columns = new_index
+    unpivoted_df = unpivoted_df.rename(
+        columns={
+            "Tab 6 - Outcomes: Section C - Footfall Indicator 1 - Relevant Project": "Project Name",
+        }
+    )
+
+    unpivoted_df = join_to_project_outcomes(unpivoted_df, df_lookup)
+
+    financial_months = [value for value in unpivoted_df.columns if re.search(r"\d{4}", value)]
+
+    # Don't want to lose any data in the melt, so get all non-melting cols
+    id_vars = unpivoted_df.columns.difference(financial_months)
+
+    # Get data for each financial month as its own row
+    unpivoted_df = pd.melt(
+        unpivoted_df,
+        id_vars=id_vars,
+        value_vars=financial_months,
+        value_name="Amount",
+        var_name="financial_period",
+    )
 
-        # unpivot table, split month cols into var column, corresponding values into val col.
-        temp_cols = pd.melt(temp_cols, id_vars=list(new_index[:7]), var_name="Month", value_name="Amount")
-        temp_cols.sort_values(["Relevant Project", "Geography"], inplace=True)
-        df_unpivot = df_unpivot.append(temp_cols)
+    unpivoted_df["financial_month"] = unpivoted_df["financial_period"].str.extract(
+        r"((?:January|February|March|April" r"|May|June|July|August|September" r"|October|November|December)\s+\d{4})"
+    )
 
-        # TODO: Some project fields are empty, some are 0, and some of these have values in. Map to programme
+    financial_month_start = pd.to_datetime(unpivoted_df["financial_month"])
+    financial_month_end = financial_month_start + pd.offsets.MonthEnd()
 
-    df_unpivot.reset_index(drop=True, inplace=True)
-    # Convert "Date" column values into datetime objects start and end dates (for the given month)
-    df_unpivot[["Start_Date", "End_Date"]] = df_unpivot["Month"].apply(lambda x: pd.Series(convert_date(x)))
-    return df_unpivot
+    unpivoted_df["Start_Date"] = financial_month_start
+    unpivoted_df["End_Date"] = financial_month_end
+
+    final_cols = [
+        "Submission ID",
+        "Project ID",
+        "Programme ID",
+        "Outcome",
+        "Start_Date",
+        "End_Date",
+        "UnitofMeasurement",
+        "GeographyIndicator",
+        "Amount",
+        "Actual/Forecast",
+        "Higher Frequency",
+    ]
+
+    unpivoted_df = unpivoted_df.rename(
+        columns={
+            "Tab 6 - Outcomes: Section C - Footfall Indicator 1 - Geography": "GeographyIndicator",
+            "Tab 6 - Outcomes: Section C - Footfall Indicator 1 - Name": "Outcome",
+            "Tab 6 - Outcomes: Section C - Footfall Indicator 1 - Unit": "UnitofMeasurement",
+        }
+    )
+
+    unpivoted_df["Actual/Forecast"] = unpivoted_df.apply(get_actual_forecast, axis=1)
+    unpivoted_df["Higher Frequency"] = np.nan
+
+    unpivoted_df = unpivoted_df[final_cols]
+
+    return unpivoted_df
```

Comment:
in the returned DF, there are custom Outcome names, but they should all be just ""Year on Year monthly % change in footfall" as pre Round 3 (after discussions with TF we assumed the footfall outcome name SHOULD always be hard-coded, and set it to the above). 

This means it can then be mapped to the correct outcome category (from the dict in const). So for this DF can you just hard code these values please? (It looks like you only have 3 projects with footfall data in the whole data extract?)